sequential/quadtree.cpp

```diff
@@ -0,0 +1,248 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+// Create the definition of a point for the QuadTree
+struct Point {
+    // Each point is defined as (x,y)
+    int x;
+    int y;
+
+    Point(int xVal = 0, int yVal = 0){
+        x = xVal;
+        y = yVal;
+    }
+};
+
+// The Nodes of a QuadTree
+struct Node {
+    Point NodePos;
+    int data;
+    
+    // Node contains coordinates and data
+    Node(Point p, int d){
+        NodePos = p;
+        data    = d;
+    }
+};
+
+// QUADTREE
+class QuadTree {
+    // Top and Bottom point coordinates for B-Box
+    Point TopLeft;
```

Comment:
Could we make the notation of a grid's boundaries consistent? Bottom Left and Top Right, just like a regular 2D axis

Reply:
Sure, currently its BottomRight and TopLeft. Only the if conditionals will change